### ISSUE TITLE
Crash in ab_glyph renderer

### ISSUE THREAD

**Issue by Friz64** [PolyMeilex/sctk-adwaita#63]
See https://github.com/pop-os/cosmic-files/issues/241

Haven't tried to reproduce it yet.

**Comment by PolyMeilex** [comment]
It's the second time this code path panics, I think I will just make it ignore out of range pixels, better to have weirdly cropped text rather than panic on every weird edge case.

**Comment by PolyMeilex** [comment]
Althou now, that I'm looking at the report, even cosmic-text can't load that font up, so maybe it's not even ab-glyphs fault: 
```
[2024-06-12T07:05:52Z WARN  cosmic_text::font::system] failed to load font 'SymbolsNFM'
````

**Comment by Friz64** [comment]
Hm, but they also said "It can show Chinese text correctly", so maybe it's an unimportant warning?

Regardless of that, I think your fix is for the better.

**Comment by PolyMeilex** [comment]
Published `0.10.1`, `0.9.1`, and `0.8.3` so it should be fine now, I believe cosmic-files is on `0.8` so cargo update should do the trick.
